PL2/PL2-core/src/main/java/nl/tudelft/pl2016gr2/core/algorithms/ZoomInGraph.java

```diff
@@ -0,0 +1,54 @@
+package nl.tudelft.pl2016gr2.core.algorithms;
+
+import nl.tudelft.pl2016gr2.model.graph.nodes.Bubble;
+import nl.tudelft.pl2016gr2.model.graph.nodes.GraphNode;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Class to zoom in on random bubbles.
+ * 
+ * @author Casper
+ *
+ */
+public class ZoomInGraph extends ZoomIn {
```

Comment:
Really weird name. This is the ZoomIn implementation for 'normal' graph bubbles?